src/main/java/us/ihmc/mecano/algorithms/JointTorqueRegressorCalculator.java

```diff
@@ -85,6 +79,11 @@ public class JointTorqueRegressorCalculator
     */
    private final JointTorqueRegressorRecursionStep[] jointTorqueRegressorRecursionSteps;
 
+   /**
+    * If computing blocks of the joint torque regressor matrix for individual bodies, this map can be used to perform body-wise computations.
+    */
+   private final Map<RigidBodyReadOnly, JointTorqueRegressorRecursionStep> rigidBodyToRecursionStepMap = new LinkedHashMap<>();
```

Comment:
Heads up here: `LinkedHashMap` allow to get a map that maintains the order in which the elements were added. This comes at a slight performance penalty when compared to the `HashMap` implementation. If you don't care about the ordering in the map, I'd recommend using a simple `HashMap`. Worth noting that in this case, I'm doubtful that you'd see a noticeable difference in speed.

Reply:
Yeah you're right. But, start as I mean to go on, I've made the change.